pom.xml

```diff
@@ -5,4 +5,24 @@
   <version>0.0.1-SNAPSHOT</version>
   <name>Fierst Java Project</name>
   <description>My first project in Java</description>
+      <build>
+        <plugins>
+            <plugin>
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <version>4.0.3</version>
+                <configuration>
+                    <url>jdbc:h2:file:./target/foobar</url>
```

Comment:
foobar to nie jest dobra na nazwę katalogu, w którym znajduje się baza danych - zmieńmy ją na portfolio-db